packages/react-ui/src/widgets/HistogramWidgetUI/HistogramWidgetUI.js

```diff
@@ -180,6 +174,15 @@ function HistogramWidgetUI({
 
   // Series
   const seriesOptions = useMemo(() => {
+    const dataWithColor = formattedData.map((item, idx) => {
+      const isDisabled = selectedBars.length && selectedBars.indexOf(idx) === -1;
```

Comment:
I'd rather go with `selectedBars.includes(idx)` instead of `selectedBars.indexOf(idx) === -1`, what do you think?

Reply:
I think that both are the same, right? Any advantages of using includes?

Reply:
It is just for readability, i think it's clearer with "includes" and you don't have to compare it with `-1`

Reply:
+1 on readability.